bin/opencollective.js

```diff
@@ -0,0 +1,41 @@
+const child_process = require("child_process");
+const chalk = require("chalk");
+
+function padding(length) {
+	var padding = "";
```

Comment:
I think we are good to use `const`

Reply:
Thanks, I replaced this function with `" ".repeat(length)` that's supported by node 4